src/core/lib/attribute-manager.js

```diff
@@ -137,8 +139,9 @@ export default class AttributeManager {
    * @param {Object} [props]
    * @param {String} [props.id] - identifier (for debugging)
    */
-  constructor({id = 'attribute-manager'} = {}) {
+  constructor({id = 'attribute-manager', gl, transition} = {}) {
```

Comment:
Nit: The convention for classes in luma.gl that take gl context is to take it as first parameter. `AttributeManager(gl, {...})`
I guess following that pattern would risk some backwards compatibility. `Model` does handle both forms.

Reply:
Note: Now that attribute manager has access to the gl context, we could have it create and manage webgl Buffers for normal attributes